src/particleContainer/AutoPasContainer.cpp

```diff
@@ -137,6 +140,15 @@ bool AutoPasContainer::rebuild(double *bBoxMin, double *bBoxMax) {
 	std::array<double, 3> boxMin{bBoxMin[0], bBoxMin[1], bBoxMin[2]};
 	std::array<double, 3> boxMax{bBoxMax[0], bBoxMax[1], bBoxMax[2]};
 
+	memcpy(_boundingBoxMin, bBoxMin, 3 * sizeof(double));
+	memcpy(_boundingBoxMax, bBoxMax, 3 * sizeof(double));
+
+	// check if autopas is already initialized
+	if(_autopasContainerIsInitialized) {
+	  _autopasContainer.resizeBox(boxMin, boxMax);
+	  return false;
+	}
+
```

Comment:
```suggestion

	// The following code is only executed if _autopasContainer has not been initialized, yet.
```
The return false is a bit "hidden". Maybe a comment can help clarify this further?